geetools/ImageCollection.py

```diff
@@ -928,21 +939,17 @@ def reduceInterval(
         # Every subcollection is sorted in case one use the "first" reducer
         imageCollectionList = self.groupInterval(unit, duration)
```

Comment:
be careful, or this package will soon be called `gee-voldemort` 🧙‍♂️

Reply:
At least I know it's there if things go crazy I'll remove it.